sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -25,6 +26,19 @@ public Discount getDiscount(@PathVariable Long id) {
         return service.getAll();
     }
 
+    //Возвращаю при пустом поле поиска или отсутсвие заданых критериев пока пустой список
+    @GetMapping("/search")
+    public List<Discount> getBySearchWord(@RequestParam String searchWord) {
+        List<Discount> result = new ArrayList<>();
```

Comment:
Please move this search logic to the service layer. Also instead of two methods `getByNameContaining` and `getByDescriptionContaining` let's create one.

Reply:
Done

Reply:
Let's leave here only one line `return service.search(searchText)`

Reply:
Ok, understand